<launch>
    <arg name="ns"/>
    <arg name="config_file_name" default="$(arg ns)"/>
    <arg name="intrinsics_file_name" default="$(arg ns)"/>
    <arg name="machine"/>

    <!-- launch camera_node -->
    <group ns="$(arg ns)">
        <node machine="$(arg machine)" pkg="camera" type="camera_node.py" name="camera_node" output="screen">
            <param name="config_file_name" value="$(arg config_file_name)"/>
            <param name="intrinsics_file_name" value="$(arg intrinsics_file_name)"/>
        </node>
    </group>


    <!-- Subscriptions -->

    <!-- Publications -->
    <!-- "~image/compressed": sensor_msgs/CompressedImage -->
    <!-- "~camera_info": sensor_msgs/CameraInfo -->
    <!-- "~camera_info_intrinsic_calibration": sensor_msgs/CameraInfo -->

</launch>
